src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import chevronNext from '../../assets/chevronNext.svg';
+
+export interface BreadcrumbItemProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Hypertext reference to navigate to breadcrumb item
+   */
+  href?: string;
+  /**
+   * Whether the user is on the current item page or not
+   */
+  isCurrent?: boolean;
+  /**
+   * Label to display in the breadcrumb
+   */
+  label?: string;
+}
+
+const BreadcrumbItem = ({ className, href, label, isCurrent = false, ...props }: BreadcrumbItemProps) => {
+  const { className: baseClassName, ...commonProps } = getCommonProps(props, 'Breadcrumb');
+  const extraClass = !isCurrent ? 'previous' : 'current';
```

Comment:
We should be following the BEM class naming convention.
```
`${baseClassName}--current`
```
We should also use classnames package to do the conditional adding of classes. 
```
className={classnames(
    baseClassName,
    className, 
   { [`${baseClassName}--current`]: isCurrent }
)}
```
And since only one can be current, do we think it make sense to have the default styles include the styles of the `previous` class and then the `current` modify can overwrite those styles?